### PR TITLE
Parse Postgres connection string function

### PR DESCRIPTION
**Pull request by neelip**

### CODE REVIEW

File: src/postgres/postgresConnectionStrings.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { parse } from "pg-connection-string";
+import { ParsedConnectionString } from "../ParsedConnectionString";
+import { nonNullProp } from "../utils/nonNull";
+
+export async function parsePostgresConnectionString(connectionString: string): Promise<ParsedPostgresConnectionString> {
```
Comment (EricJizbaMSFT) [inline src/postgres/postgresConnectionStrings.ts:10]: What's the purpose of adding this function/file? I prefer to see how something is used when it's added

Comment (neelip) [reply src/postgres/postgresConnectionStrings.ts:10]: This is to parse the postgres connection string for attached accounts support. The latest PR for postgres attached accounts includes this commit.

**Comment by nturinski** [comment]
Does this still need review?  It seems like this PR https://github.com/microsoft/vscode-cosmosdb/pull/1640 has code that is more up to date, so I assume that we can close this one?

**Comment by neelip** [comment]
Sounds good! This PR doesn't contain the latest code.

**Pull request closed without merging**